_posts/2025-05-28-#32051.md

```diff
@@ -7,25 +7,102 @@ authors: [jonatack]
 components: ["p2p"]
 host: jonatack
 status: upcoming
-commit:
+commit: 93b07997e9a38523f5ab850aa32ca57983fd2552
 ---
 
-_Notes and questions to follow soon!_
-
-<!-- TODO: Before meeting, add notes and questions
 ## Notes
 
+### Motivation
+
+While doing initial block download (IBD) over a fluctuating and slow internet
+connection in El Salvador, I observed very frequent peer disconnections in the
+debug log, on the order of 100+ per hour. These disconnections were often of
+manually added "addnode" peers, and logged as `Peer is stalling block download,
```

Comment:
I gave this log output its own line to make it easier to readers to copy and git grep to find it in the codebase.

If this is brought into one paragraph, would wrap it in quotes rather than code backticks.

Reply:
It's rendered as one line anyway so easy enough for people to copy it!